components/tfx/README.md

```diff
@@ -0,0 +1,17 @@
+## TFX components for Kubefow Pipelines
+
+Disclaimer: The TFX components in this directory are unofficial and are maintained by the KFP team not the TFX team.
+
+If you experience any issues in this components please create a new issue in the Kubeflow Pipelines repo and assign it to Ark-kun.
+
+These components were created to allow the users to use TFX components in their KFP pipelines, to be able to mix KFP and TFX components.
+
+If your pipeline uses only TFX components, please use the official [TFX SDK](https://www.tensorflow.org/tfx/guide).
```

Comment:
Perhaps point the user to https://www.tensorflow.org/tfx/tutorials/tfx/cloud-ai-platform-pipelines ?

Took a look and I think most of the content at https://www.tensorflow.org/tfx/guide are related to TFX components/libraries but not pipeline authoring. Up to you

Reply:
Done.